python/ray/actor.py

```diff
@@ -1817,3 +1914,42 @@ def exit_actor():
             f"{worker.mode}. Call this API inside an actor methods"
             "if you'd like to exit the actor gracefully."
         )
+
+
+def _handle_tensor_transport(dst_actor, args):
+    """Handle tensor transport between actors.
```

Comment:
"Handle tensor transport" is a bit vague.

Suggested: Trigger tensor communication operation between actors. This is a helper function to be called when an ObjectRef containing an in-actor tensor is passed to another actor task, and the data should be passed out-of-band instead of through the object store.

Reply:
Updated https://github.com/ray-project/ray/pull/52938/commits/dd735db476e6c339aecdb58641077704476ec4c4